### DISCUSSION TITLE
Missing mock.module in [@types/node]

### DISCUSSION THREAD

**Discussion by rozzilla** [DefinitelyTyped/DefinitelyTyped#69837] [Issues with a @types package]
Hi everyone! 👋🏼 
The [mock.module](https://nodejs.org/api/test.html#mockmodulespecifier-options) is available on Node v22.
So it'd be great to have it in `@types/node`.
Considering it's a v22-only feature, how does it work in such cases?

By the way, I'll happy to work on it if nobody else want to pick it up this ticket 😉

[a comment by an automated account is not shown]

**Comment by Anonymous4078** [comment]
Nobody has started work on types node v22
